rtengine/iptransform.cc

```diff
@@ -546,44 +604,47 @@ static float calcPCVignetteFactor(const struct pcv_params& pcv, int x, int y)
     }
 
     if (pcv.is_super_ellipse_mode) {
-        float dist_oe1 = pcv.oe1_a * pcv.oe1_b / pow_F(pow(pcv.oe1_b * sincosval.y, pcv.sep) + pow(pcv.oe1_a * sincosval.x, pcv.sep), 1.0 / pcv.sep);
-        float dist_oe2 = pcv.oe2_a * pcv.oe2_b / pow_F(pow(pcv.oe2_b * sincosval.y, pcv.sep + 2) + pow(pcv.oe2_a * sincosval.x, pcv.sep + 2), 1.0 / (pcv.sep + 2));
-        float dist_ie1 = pcv.ie1_mul * dist_oe1 * (1.0 - pcv.feather);
-        float dist_ie2 = pcv.ie2_mul * dist_oe2 * (1.0 - pcv.feather);
-        dist_oe = dist_oe1 * (1.0 - pcv.sepmix) + dist_oe2 * pcv.sepmix;
-        dist_ie = dist_ie1 * (1.0 - pcv.sepmix) + dist_ie2 * pcv.sepmix;
+        float dist_oe1 = pcv.oe1_a * pcv.oe1_b / normn(pcv.oe1_b * sincosval.y, pcv.oe1_a * sincosval.x, pcv.sep);
+        float dist_oe2 = pcv.oe2_a * pcv.oe2_b / normn(pcv.oe2_b * sincosval.y, pcv.oe2_a * sincosval.x, pcv.sep + 2);
+        float dist_ie1 = pcv.ie1_mul * dist_oe1;
+        float dist_ie2 = pcv.ie2_mul * dist_oe2;
+        dist_oe = dist_oe1 * (1.f - pcv.sepmix) + dist_oe2 * pcv.sepmix;
+        dist_ie = dist_ie1 * (1.f - pcv.sepmix) + dist_ie2 * pcv.sepmix;
     } else {
         dist_oe = pcv.oe_a * pcv.oe_b / sqrtf(SQR(pcv.oe_b * sincosval.y) + SQR(pcv.oe_a * sincosval.x));
-        dist_ie = pcv.ie_mul * dist_oe * (1.0 - pcv.feather);
+        dist_ie = pcv.ie_mul * dist_oe;
     }
 
     if (dist <= dist_ie) {
```

Comment:
Is this still true? I mean, `dist_ie` is no longer scaled by `(1.0 - pcv.feather)`, whereas `dist` seems unchanged.